lightly/api/download.py

```diff
@@ -68,10 +73,32 @@ def download_all_video_frames(
 
     """
     _check_av_available()
+    timestamp = 0 if timestamp is None else timestamp
+    if timestamp < 0:
+        raise ValueError(f"Negative timestamp is not allowed: {timestamp}")
```

Comment:
I am actually not sure if negative timestamps are not allowed. I did not add the check because we only use the function with timestamps we already retrieved from the frames, so they should be correct.